.gitignore

```diff
@@ -1,3 +1,4 @@
 build
 composer.lock
-vendor
\ No newline at end of file
+vendor
+.DS_STORE
```

Comment:
It's generally discouraged, because where does it stop? You could end up with a VERY long .gitignore file in every project. Generally it's recommended to use a global .gitignore file isntead for this sort of stuff.